tests/anonymous-class-attribute.phpt

```diff
@@ -2,9 +2,8 @@
 Attribute declared in anonymous class
 
 --SKIPIF--
-<?php version_compare(PHP_VERSION, "8.0", ">=")
-      or die("skip because attributes are only available since PHP 8.0") ?>
-
+<?php
+if (!version_compare(PHP_VERSION, "8.0", ">=")) echo "skip because attributes are only available since PHP 8.0";
```

Comment:
```suggestion
if (!version_compare(PHP_VERSION, "8.0", ">=")) {
    echo "skip because attributes are only available since PHP 8.0";
}
```

As the `SKIPIF` is no longer a conditional expression with `or`, might be good to use a little cleaner code style for the code. (here and elsewhere)